test/contracts/common/time_helpers.js

```diff
@@ -7,17 +9,18 @@ contract('TimeHelpers', () => {
 
   it('checks block number', async () => {
     assert.equal((await timeHelpersMock.getBlockNumberExt()).toString(), (await timeHelpersMock.getBlockNumber64Ext()).toString(), "block numbers should match")
-    assert.equal((await timeHelpersMock.getBlockNumberExt()).toString(), (await timeHelpersMock.getBlockNumberDirect()).toString(), web3.eth.blockNumber, "block number should match with real one", "block number should match with real one")
+    assert.equal((await timeHelpersMock.getBlockNumberExt()).toString(), (await timeHelpersMock.getBlockNumberDirect()).toString(), web3.eth.getBlockNumber, "block number should match with real one", "block number should match with real one")
```

Comment:
A bit confused with this (although it's more with the original test)—do we not need to call `getBlockNumber()`?

Reply:
I think the original was malformed, let me change it